python/torch_mlir_e2e_test/test_suite/norm_like.py

```diff
@@ -332,3 +332,23 @@ def forward(self, x):
 @register_test_case(module_factory=lambda: LayerNormNormalizeOverAllDimsModule())
 def LayerNormNormalizeOverAllDimsModule_basic(module, tu: TestUtils):
     module.forward(tu.rand(2, 2, 3))
+
+# ==============================================================================
+
+class LayerNormScalarOptDimModule(torch.nn.Module):
+    def __init__(self):
+        super().__init__()
+
+    @export
+    @annotate_args([
+        None,
+        ([-1, -1, -1, -1], torch.float32, True),
+    ])
+    def forward(self, x):
+        #return torch.ops.aten.norm.ScalarOpt_dim(x, p = 2, dim = 2, keepdim = True)
+        return torch.ops.aten.norm.ScalarOpt_dim(x, p = 2, dim = 3)
+
+
+@register_test_case(module_factory=lambda: LayerNormScalarOptDimModule())
+def LayerNormScalarOptDimModule_basic(module, tu: TestUtils):
+    module.forward(tu.rand(1, 1, 1, 1))
```

Comment:
can you make a tensor with more than one element? Otherwise the summation aspect of this op is not really getting tested